gtsam/geometry/PinholeCamera.h

```diff
@@ -312,6 +312,17 @@ class GTSAM_EXPORT PinholeCamera: public PinholeBaseK<Calibration> {
     return range(camera.pose(), Dcamera, Dother);
   }
 
+  /// for Linear Triangulation
+  Matrix34 getCameraProjectionMatrix() const {
+    Matrix34 P = Matrix34(PinholeBase::pose().inverse().matrix().block(0, 0, 3, 4));
```

Comment:
Avoid intermediate, just mult with block in return statement (without Matrix34 even)

Reply:
done